### PR TITLE
phpunit 6

### PR DESCRIPTION
**Pull request by kokspflanze**
drop php 5.5
added phpunit 6 support
travis update

if that is merged i will do the `::class` job

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -25,18 +25,18 @@
         "doctrine/annotations": "~1.0",
         "zendframework/zend-cache": "^2.6.1",
         "zendframework/zend-captcha": "^2.7.1",
-        "zendframework/zend-code": "^2.6 || ^3.0",
+        "zendframework/zend-code": "^2.6",
```
Comment (froschdesign) [inline composer.json]: Why the lower version?

Comment (kokspflanze) [inline composer.json]: wtf, thanks, missed smth in my tests

File: composer.json [hunk 2]
```diff
@@ -25,12 +25,12 @@
         "doctrine/annotations": "~1.0",
         "zendframework/zend-cache": "^2.6.1",
         "zendframework/zend-captcha": "^2.7.1",
-        "zendframework/zend-code": "^2.6",
+        "zendframework/zend-code": "^2.6 || ^3.0",
```
Comment (froschdesign) [inline composer.json]: Sorry, again. 😉 

Why is the lower one needed?

Comment (kokspflanze) [inline composer.json]: you mean to drop the old 2.x parts if a v3 exist? i would create a new PR for that

**Pull request closed without merging**